packages/cli/src/util/link/setup-and-link.ts

```diff
@@ -216,7 +216,8 @@ export default async function setupAndLink(
         output,
         projectSettings,
         framework,
-        autoConfirm
+        autoConfirm,
+        null
```

Comment:
Do we need to consider line 183 too?

https://github.com/vercel/vercel/blob/7d5d63965ecb15a888b8ed407fc72d89a1a91737/packages/cli/src/util/link/setup-and-link.ts#L183

Reply:
This block is only executed when `isZeroConfig` is `true`. The expression for `isZeroConfig` starts with `!localConfig ||`, and so if I understand it correctly, we'd never reach line 183 if a local configuration is provided. In fact, we'd never even create a new deployment if a local configuration exists (the deployment block is wrapped in an if conditional using `isZeroConfig`

However,

Reply:
Ah i had the short circuiting logic backwards. Okay let me dig deep here